corehq/apps/app_manager/views/modules.py

```diff
@@ -728,14 +729,27 @@ def edit_report_module(request, domain, app_id, module_id):
     provided in the request. Components are short, long, filter, parent_select,
     and sort_elements.
     """
+    def insert_prefilters(spec):
+        """
+        PreFilters are dropped from the view context when report configs are sent to the browser (cf.
+        `_get_report_module_context()`). They need to be reinserted.
```

Comment:
Ok cool. I'll give you a bit of time to explore that alternative and report back here if you want to change it.